advec1d.py

```diff
@@ -0,0 +1,97 @@
+'''
+Copyright (C) 2017-2019  Waterloo Quantitative Consulting Group, Inc.
+See COPYING and LICENSE files at project root for more details.
+'''
+
+import numpy as np
+from lib import pyblitzdg
+import matplotlib.pyplot as plt
+
+def advec1dComputeRHS(u,c, nodes1d):
+    vmapM = nodes1d.vmapM
+    vmapP = nodes1d.vmapP
+    mapI = nodes1d.mapI
+    mapO = nodes1d.mapO
+    nx = nodes1d.nx
+    rx = nodes1d.rx
+    Dr = nodes1d.Dr
+    Lift = nodes1d.Lift
+    Fscale = nodes1d.Fscale
+
+    uC = np.squeeze(np.reshape(u, (u.shape[0]*u.shape[1],1), order='F'), axis=1)
+    nxC = np.squeeze(np.reshape(nx, (nx.shape[0]*nx.shape[1],1), order='F'), axis=1)
```

Comment:
This seems complicated (error prone?) - if we can find a better way to do this then we'll scare off fewer users

Reply:
Yeah, we could add a helper that does this. Basically all it's doing is reshaping from a 2D tensor to a 1D array and traversing the tensor in column-major (Fortran-style) order.